src/cdk-experimental/listbox/listbox.spec.ts

```diff
@@ -655,3 +738,23 @@ class ListboxActiveDescendant {
     this.focusedOption = option;
   }
 }
+
```

Comment:
I would also add test cases for actually using the listbox with a `FormControl`, and then testing that the interaction works as expected via that `FormControl`